src/main/scala/guessbook/GuessBook.scala

```diff
@@ -0,0 +1,152 @@
+package guessbook
+
+import guesswho.GenderEnum
+
+import scala.util.Random
+
+object GuessBook {
+  // instantiate all books
+  val TheHonjinMurders: Book = Book(title = "The Honjin Murders", attributes = Map(CountryOfOrigin -> Country.Japan, Honkaku -> true, AuthorGender -> GenderEnum.Male,
+    SchoolOrUniversityStudents -> false, DetectiveType -> Detective.PrivateProfessional, Series -> true, PushkinVertigo -> true, SuspectPoolType -> SuspectPool.RestrictedCommunity))
```

Comment:
an extension task (from Aashvin) would be to use a csv (or any other) file that would hold all this data, and could be parsed dynamically into books at the start of the programme

Reply:
Created method getBooksFromCSV() to do this